static/js/publisher/release/revisionsTable.js

```diff
@@ -307,29 +311,25 @@ export default class RevisionsTable extends Component {
   }
 
   render() {
-    const { releasedChannels, archs, tracks } = this.props;
-
+    let { releasedChannels, archs, tracks } = this.props;
+    //archs = archs.concat(archs).concat(archs).concat(['test']);
```

Comment:
should probably remove this if it was for testing while dev'ing.

Reply:
Damn, every time I was explicitly skipping it from the commit I knew that there will be this one time when I'm gonna commit it.